app/controllers/miq_task_controller.rb

```diff
@@ -394,32 +385,40 @@ def build_query_for_warn
   end
 
   def build_query_for_status_completed(status)
-    return ["(state=? AND status=?)", ["finished", status]] if vm_analysis_task?
-    ["(state=? AND status=?)", ["Finished", status.capitalize]]
+    return ["(#{db_table}state=? AND #{db_table}status=?)", ["finished", status]] if vm_analysis_task?
+    ["(#{db_table}state=? AND #{db_table}status=?)", ["Finished", status.capitalize]]
   end
 
   def build_query_for_running
-    return ["(state!=? AND state!=? AND state!=?)", %w(finished waiting_to_start queued)] if vm_analysis_task?
-    ["(state!=? AND state!=? AND state!=?)", %w(Finished waiting_to_start Queued)]
+    sql = "(#{db_table}state!=? AND #{db_table}state!=? AND #{db_table}state!=?)"
+    if vm_analysis_task?
+      [sql, %w(finished waiting_to_start queued)]
+    else
+      [sql, %w(Finished waiting_to_start Queued)]
```

Comment:
dynamic SQL generated against 2 different tables: ```jobs``` table (for tabs ```tasks_1``` and ```tasks_3```)  and ```miq_tasks``` table (for tabs ```task_2``` and ```tasks_4```)
 ```state``` column present in both tables, but values are different: in ```jobs``` table there are starting with low case and in ```miq_tasks``` with with upper case. List of values to populate state filter for ```jobs``` table: https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/helpers/ui_constants.rb#L410
and for ```miq_tasks``` table: https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/helpers/ui_constants.rb#L415

There is ```waiting_to_start``` value in downcase which should not be there (https://github.com/ManageIQ/manageiq-ui-classic/blob/master/app/controllers/miq_task_controller.rb#L403), and may be we should delete it,  however it is there from the beginning and I am not sure if it is some legitimate exception or old bug.